### PR TITLE
Changing name endpoints for perf automation

### PR DESCRIPTION
**Pull request by nathan-weinberg**
In response to changes made in [NETOBSERV-373](https://issues.redhat.com/browse/NETOBSERV-373) and [NETOBSERV-433](https://issues.redhat.com/browse/NETOBSERV-433)

### CODE REVIEW

File: scripts/console-plugin-patch.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   {
     "op": "REPLACE_CONSOLE_PLUGIN_OPS",
```
Comment (memodi) [inline scripts/console-plugin-patch.json:3]: I think we can safely remove this file,  this patch is no longer being used.

File: scripts/flows_v1alpha1_flowcollector_lokistack.yaml [hunk 2]
```diff
@@ -78,7 +78,7 @@ spec:
       insecureSkipVerify: true
   consolePlugin:
     register: true
-    image: 'quay.io/netobserv/network-observability-console-plugin:main'
+    image: 'quay.io/netobserv/netobserv-console-plugin:main'
```
Comment (memodi) [inline scripts/flows_v1alpha1_flowcollector_lokistack.yaml:81]: this isn't changing quay and github repo names are staying as is.

File: scripts/flows_v1alpha1_flowcollector_versioned_lokistack.yaml [hunk 3]
```diff
@@ -66,7 +66,7 @@ spec:
       insecureSkipVerify: true
   consolePlugin:
     register: true
-    image: 'quay.io/netobserv/network-observability-console-plugin:v0.1.4'
+    image: 'quay.io/netobserv/netobserv-console-plugin:v0.1.4'
```
Comment (memodi) [inline scripts/flows_v1alpha1_flowcollector_versioned_lokistack.yaml:69]: this isn't changing quay and github repo names are staying as is.

File: scripts/netobserv-catalogsource.yaml [hunk 4]
```diff
@@ -5,5 +5,5 @@ metadata:
   namespace: openshift-marketplace
 spec:
   displayName: NetObservTest
-  image: 'quay.io/netobserv/network-observability-operator-catalog:vmain'
+  image: 'quay.io/netobserv/netobserv-operator-catalog:vmain'
```
Comment (memodi) [inline scripts/netobserv-catalogsource.yaml:8]: this isn't changing quay and github repo names are staying as is.

**Review by memodi — CHANGES_REQUESTED** [review]
quay/github repo names are staying same and some other comments. Also, could you add success run probably along with short workload just so we can have e2e validation along with nope.py. 
Thanks!

File: scripts/flows_v1alpha1_flowcollector_lokistack.yaml (hunk 2 above)
Comment (jotak) [inline scripts/flows_v1alpha1_flowcollector_lokistack.yaml:81]: the quay images remain the same, that's still "network-observability-console-plugin"

File: scripts/flows_v1alpha1_flowcollector_versioned_lokistack.yaml (hunk 3 above)
Comment (jotak) [inline scripts/flows_v1alpha1_flowcollector_versioned_lokistack.yaml:69]: same here

File: scripts/netobserv-catalogsource.yaml (hunk 4 above)
Comment (jotak) [inline scripts/netobserv-catalogsource.yaml:8]: same here

**Pull request merged**